### ISSUE TITLE
Failed to install ...stable: Wrong size for extra data

### ISSUE THREAD

**Issue by chang-zhao** [flathub/com.jagex.RuneScape#160]
Installation fails with an error:

```
Installing: com.jagex.RuneScape/x86_64/stable from flathub
[####################] Downloading files: 16/16 3,9 MB (1,3 MB/s)               
error: Failed to install com.jagex.RuneScape/x86_64/stable: Wrong size for extra data 
https://content.runescape.com/downloads/ubuntu/pool/non-free/r/runescape-launcher/runescape-launcher_2.2.9_amd64.deb```

**Comment by chang-zhao** [comment]
I found a similar issue, but it was closed, no idea why:
https://github.com/flathub/com.jagex.RuneScape/issues/13#issue-305625090

**Comment by Garbee** [comment]
Fixed with MR #159 ... Interesting the bot stopped auto-merging.
